src/pages/profile/index.tsx

```diff
@@ -0,0 +1,158 @@
+  },
+  [theme.breakpoints.up('md')]: {
+    fontSize: '1.5rem',
+  },
+};
+
+interface IProfileProps {}
+
+const Profile: React.FunctionComponent<IProfileProps> = (props) => {
+  return (
+    <Container maxWidth="md">
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          width: 1,
+          height: 'calc(100vh - 65.3px)',
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'flex-end',
+            flexDirection: 'column',
+            width: 1,
+            height: '500px',
+            backgroundColor: '#e8eaf6',
+            borderRadius: '33px',
+            mt: '200px',
+            mb: '40px',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'center',
+              alignItems: 'center',
+              mb: '50px',
+            }}
+          >
+            <Avatar
+              alt="Remy Sharp"
+              srcSet="https://citaty.info/files/styles/poster/public/characters/8604.jpg?itok=_bemMaL1"
+              sx={{
+                width: { sm: '250px', md: '270px', xs: '200px' },
+                height: { sm: '250px', md: '270px', xs: '200px' },
+              }}
+            />
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              width: 1,
+              alignItems: 'center',
+            }}
+          >
+            <Typography variant="h4" color="initial">
+              Harry Potter
+            </Typography>
+            <Typography variant="subtitle1" color="#696969">
+              Wizard
+            </Typography>
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                p: '10px',
+                mb: '40px',
+                backgroundColor: '#fcfefe',
+                borderRadius: '33px',
+              }}
+            >
+              <Box
+                sx={{
+                  pr: '20px',
+                  textAlign: 'right',
+                  fontFamily: 'roboto',
+                }}
+              >
+                <ThemeProvider theme={theme}>
+                  <Typography
+                    variant="h6"
+                    color="#696969"
+                    sx={{
+                      pt: '15px',
+                    }}
+                  >
+                    City
+                  </Typography>
+                  <Typography variant="h6" color="#696969">
+                    Country
+                  </Typography>
+                  <Typography variant="h6" color="#696969">
+                    DoB
+                  </Typography>
+                  <Typography variant="h6" color="#696969">
+                    Email
+                  </Typography>
+                  <Typography variant="h6" color="#696969">
```

Comment:
use colours from theme, not hardcoded values

Reply:
fixed